README.md

```diff
@@ -78,3 +78,32 @@ start the Bee cluster
 ```sh
 $ ./scripts/environment.sh start
 ```
+
+## Utilities
+
+It is possible to generate random traffic in your cluster:
+
+```sh
+$ npm run gen:traffic 
+```
+
+The script is in an infinite loop, so if you want to stop the generation you have to terminate it manually in your terminal by pressing `Ctrl^C`.
+
+If you don't specify any parameters it will produce 400 chunks/0.5 sec that the script tries to upload on the `http://localhost:1633` - that is the binded port of the queen node if you orchestrated the environment with the `envrionment.sh`.
+
+The following way you can pass parameters
+
+1. HOST - Host that has reachable port to the Bee API. [string,Default:'localhost']
+2. SECURE_CONNECTION - whether the connection is HTTPS. [boolean,Default:'false']
+3. BEE_SH_PORTS - the traffic generation will try to go through all possible Bee node binding on the host (bee.sh port binding). [boolean,Default:'false']
```

Comment:
This is unclear to me what it does. Does it mean it iterates through ports `1633`, `11633`, `21633` ...? My suggestion would be to just scratch all this and accept single array which lists all the URLs where Bee is running and which user want to use in the script. So that would be: `["http://localhost:1633", "http://localhost:11633"]`

Reply:
ur suggestion was better than the current use, so I implemented, check it out!